scripts/Jenkinsfiles/python

```diff
@@ -168,14 +168,15 @@ pipeline {
                         unstash 'commonlib-unit-reports'
                         sh """export CI_BRANCH=$ci_branch
                         export TARGET_BRANCH=$target_branch
+                        export TEST_SUITE=python
```

Comment:
^ what's going on here? You are exporting, but no longer using it below

Reply:
It's used by the upload command. Currently, that command is trying to upload `None.coverage`, which isn't very useful.

I changed the lower one just to make sure all coverage files got archived.